src/olympia/devhub/views.py

```diff
@@ -1690,22 +1707,35 @@ def api_key_agreement(request):
     return render_agreement(request, 'devhub/api/agreement.html', next_step)
 
 
-def render_agreement(request, template, next_step):
-    if request.method == 'POST':
+def render_agreement(request, template, next_step, **extra_context):
+    new_style_agreement = waffle.switch_is_active('post-review')
+    form_class = AgreementForm if new_style_agreement else Form
```

Comment:
Can you add a comment explaining what `Form` is for?  I was scratching my head at first what form this badly named `Form` was and what it did.